composer.json

```diff
@@ -13,8 +13,8 @@
         "doctrine/common": "^2.7",
         "doctrine/data-fixtures": "^1.2",
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
-        "doctrine/phpcr-bundle": "^2.0@dev",
-        "doctrine/phpcr-odm": "^1.4 || ^2.0@dev",
+        "doctrine/phpcr-bundle": "^1.4 || ^2.0",
```

Comment:
i am not sure that phpcr-bundle 1.4 will work again with this.

Reply:
It is just a door opener for the other components to use it as they use symfony from 2.8 to 4.0